nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -314,3 +316,54 @@ def list_available_models(cls) -> Optional[Dict[str, str]]:
     @classmethod
     def from_pretrained(cls, name: str):
         pass
+
+    def _prepare_for_export(self):
+        return self.bert_model._prepare_for_export()
+
+    def export(
+        self,
+        output: str,
+        input_example=None,
```

Comment:
this input_example and output_example are for Model, right?

Reply:
Enforced recomputing both